SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -4205,8 +4205,16 @@ class CSolver {
    * \param[in] config - Definition of the particular problem.
    * \param[in] referenceCoord - Determine if the mesh is deformed from the reference or from the current coordinates.
    */
-  inline virtual void SetMesh_Stiffness(CGeometry **geometry,
-                                        CNumerics **numerics,
+  inline virtual void DeformMesh(CGeometry *geometry,
+                                 CNumerics **numerics,
+                                 CConfig *config) { }
+
+  /*!
+   * \brief A virtual member.
+   * \param[in] config - Definition of the particular problem.
+   * \param[in] referenceCoord - Determine if the mesh is deformed from the reference or from the current coordinates.
+   */
+  inline virtual void SetMesh_Stiffness(CNumerics **numerics,
```

Comment:
Here, a `CGeometry**` was expected, but not used. To make the function compatible with `CDeformationDriver` which does not have multigrid, this argument was removed (at the cost of some consistency with the signatures of other methods, maybe).